app/models/vote.rb

```diff
@@ -82,5 +82,11 @@ def evaluate_reward(new_vote, value, subject)
       end
       reward
     end
+
+    def voted(resource, user)
+      vote = where(user_id: user.id, voteable_id: resource.id, voteable_type: resource.class.to_s).first
```

Comment:
What do think of giving **votes** a default value of **0**, then we could just return the query result?
We can then modify this method to 
```ruby
select(:value).where(user: user, votable: resource).first
```
or something even better :wink: